qutip/qobjevo_codegen.py

```diff
@@ -38,12 +38,48 @@
 import os
 import numpy as np
 from qutip.cy.inter import _prep_cubic_spline
+import time
+
+
+def _try_remove(filename):
+    try:
+        os.remove(filename)
+    except Exception:
+        pass
+
+
+def import_str(code, basefilename, obj_name, cythonfile=False):
```

Comment:
Can you add docstrings?

Reply:
I made a few new functions to make codeclimat happy: small simple function, little repeated code.